tests/test_events.py

```diff
@@ -104,3 +151,131 @@ def test_publish(self):
             course=block.location.course,
             run=block.location.run,
         )
+
+    @data(*[
+        ['choice_0', 'an incorrect answer'],
+        ['choice_1', 'the correct answer'],
+        ['choice_2', 'a different incorrect answer'],
+    ])
```

Comment:
What exactly are we testing w/ these 3 different test cases? Seems like just submitting 1 answer would cover everything that we care about

Reply:
I wanted to make sure in each case that we record the right answer. There is also a `student_answers` dict which may contain a different answer than the one the student actually chose